### ISSUE TITLE
Bug: path to shortcut.vbs might contain lib/windows-i686 in some cases

### ISSUE THREAD

**Issue by dreamscached** [Friends-of-Monika/mas-autostart#4]
### What happened?

(Reported by another person in Discord) 

Their install of MAS Autostart Mod is unable to invoke shortcut.vbs script due to invalid path.

### Log or traceback output

```
[2022-08-13 11:25:32] [!ERROR! T_T]: [MAS Autostart Mod] Got non-zero exit code from shortcut script invocation (1.)
[2022-08-13 11:25:32] [DEBUG]: [MAS Autostart Mod] shortcut.vbs was called with parameters: (u'cscript', u'/nologo', u'C:\\Users\\XXX\\Everything\\Game stuff\\Other, unclassified games\\Just Monika\\lib\\windows-i686\\Submods\\MAS Autostart Mod\\platform\\shortcut.vbs', u'C:\\Users\\XXX\\AppData\\Roaming\\Microsoft\\Windows\\Start Menu\\Programs\\Startup\\Monika After Story.lnk', u'C:\\Users\\XXX\\Everything\\Game stuff\\Other, unclassified games\\Just Monika\\DDLC.exe', u'C:\\Users\\XXX\\Everything\\Game stuff\\Other, unclassified games\\Just Monika').
```

### Submod version

1.1.6

### Monika After Story version

0.12.9

### Unstable

No

### Code of Conduct and Contribution Guidelines

- [X] I agree to follow this project's Code of Conduct and Contribution Guidelines

**Comment by dreamscached** [comment]
Additionally, this happened on startup:

```
While running game code:
  File "game/Submods/MAS Autostart Mod/topics.rpy", line 98, in <module>
    $ store.masAutostart_api.disable()
  File "game/Submods/MAS Autostart Mod/autostart.rpy", line 432, in disable
    _disable_delete_desktop_file()
  File "game/Submods/MAS Autostart Mod/autostart.rpy", line 445, in _disable_delete_desktop_file
    os.remove(persistent._masAutostart_metadata[1])
TypeError: 'NoneType' object has no attribute '__getitem__'
```

**Comment by dreamscached** [comment]
It is unclear what causes the issue, as attempts to install this submod cleanly on new fresh DDLC+MAS install were successful and it worked.

**Comment by dreamscached** [comment]
Additional debug info:
```
[2022-08-13 11:25:31] [DEBUG]: [MAS Autostart Mod] autostart dir: C:\Users\XXX\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Startup, launcher path: C:\Users\XXX\Everything\Game stuff\Other, unclassified games\Just Monika\DDLC.exe, default autostart file: C:\Users\XXX\AppData\Roaming\Microsoft\Windows\Start Menu\Programs\Startup\Monika After Story.lnk, autostart shortcut script: C:\Users\XXX\Everything\Game stuff\Other, unclassified games\Just Monika\lib\windows-i686\Submods\MAS Autostart Mod\platform\shortcut.vbs
```

**Comment by dreamscached** [comment]
Fixed in dadaf247168b93af49368025f2751b7df935a00d.
